sequence_classification/datasets/sentiment/__init__.py

```diff
@@ -7,10 +7,7 @@ class CommentSentimentDataset(DatasetLoader):
 
     @staticmethod
     def load_data(name='Comment Sentiment'):
-        data = []
-        for file in ['amazon_cells_labelled.txt', 'imdb_labelled.txt', 'yelp_labelled.txt']:
-            data.append(np.genfromtxt(CommentSentimentDataset.get_dataset_file(['sentiment', file]),
-                                      delimiter='\t', encoding="utf-8", dtype=None))
-        sentiment_data = np.concatenate(tuple(data), axis=0)
-        X, y = get_X_y(sentiment_data)
+        data = np.genfromtxt(CommentSentimentDataset.get_dataset_file(['sentiment', 'amazon_cells_labelled.txt']),
```

Comment:
Skoro w jednym pliku połączyliście 3 zbiory danych, to dobrze by było zmienić temu plikowi nazwę (co najmniej usunąć z nazwy amazon)